pulsar-client-cpp/tests/HashTest.cc

```diff
@@ -58,10 +58,18 @@ TEST(HashTest, testJavaStringHash) {
 
 TEST(HashTest, testMurmur3_32Hash) {
     Murmur3_32Hash hash;
+    std::string k1 = "k1";
+    std::string k2 = "k2";
     std::string key1 = "key1";
     std::string key2 = "key2";
+    std::string key01 = "key01";
+    std::string key02 = "key02";
 
     // Same value as Java client
+    ASSERT_EQ(2110152746, hash.makeHash(k1));
+    ASSERT_EQ(1479966664, hash.makeHash(k2));
```

Comment:
What is the purpose of adding these test cases(k1, k2, key01, key02)?
If you want to compare more values with Java client, I think you should add the same test cases to [HashTest.java](https://github.com/apache/incubator-pulsar/blob/8d159efa5a4f25191fade5a9fddb45a245ef2e6b/pulsar-client/src/test/java/org/apache/pulsar/client/impl/HashTest.java#L41-L48) ?

Reply:
That's because this problem occurs when the key length is not a multiple of 4.
We need to test if the hashes generated from keys whose length is not a multiple of 4 are correct.

Reply:
> We need to test if the hashes generated from keys whose length is not a multiple of 4 are correct.

I got it, thanks.